python_scripts/xx_basic_parameters_tuning.py

```diff
@@ -119,32 +119,34 @@
 # ## Finding the best model hyper-parameters via exhaustive parameters search
 #
 # We see that the parameter `C` as a significative impact on the model
-# performance and that finding the best value for this parameter is crucial.
-# However, this parameter should be tuned with cross-validation such that
-# we find a parameter. In short, we will set the parameter, train our model
-# on some data, and evaluate the performance of our model on some left out
-# data. Ideally, we will select the parameter leading to the optimal
-# performance on the testing set. Scikit-learn provides a `GridSearchCV`
-# estimator which will handle the cross-validation for us.
+# performance. Finding the best value for this parameter is crucial.
+# However, this parameter should be tuned with cross-validation, avoinding
```

Comment:
This parameter should be tuned to get the best cross-validation score, so as to avoid
over-fitting problems.